core/arch/arm/kernel/unwind_arm32.c

```diff
@@ -367,10 +367,14 @@ void print_stack(int level)
 {
 	struct unwind_state state;
 
-	memset(&state, 0, sizeof(state));
+	memset(state.registers, 0, sizeof(state.registers));
```

Comment:
Because, that's the only part that contains input arguments. The rest is used internally by the unwinding code, so it doesn't look like it is the responsibility of the caller to initialize it (To which value BTW? Is zero acceptable for all fields?).